controllers/host/host_controller.go

```diff
@@ -1383,6 +1385,42 @@ func (r *HostReconciler) ReconcileExistingHost(client *gophercloud.ServiceClient
 
 	logHost.Info("current config is:", "values", current)
 
+	specBytes, err := json.Marshal(profile)
```

Comment:
If marshal fails, can we refer the specBytes value?

Reply:
Refactored. Thanks!